receiver/statsdreceiver/protocol/statsd_parser.go

```diff
@@ -388,3 +417,7 @@ func parseMessageToMetric(line string, enableMetricType bool) (statsDMetric, err
 
 	return result, nil
 }
+
+func addrToKey(addr net.Addr) string {
+	return addr.Network() + "_" + addr.String()
+}
```

Comment:
Would you create a `type netAddr struct { Network, String string }` and use it as in `map[netAddr]*instruments` with `netAddr{ Network: addr.Network(), String: addr.String() }` as the key , as opposed to the string concatenation done here?

Reply:
done